### PR TITLE
Created global success toaster.

### PR DESCRIPTION
**Pull request by michal-damiecki**
UIE-352

### CODE REVIEW

File: src/SuccessToaster/SuccessToaster.jsx [hunk 1]
```diff
@@ -75,13 +67,20 @@ export class SuccessToaster extends Component {
...
 }
 
-export default SuccessToaster;
+const mapStateToProps = state => ({
+  data: getData(state),
+  title: getTitle(state),
+});
+
+export default connect(mapStateToProps, {
+  clearResponse,
```
Comment (stepek) [inline src/SuccessToaster/SuccessToaster.jsx]: Please rename `clearResponse` to something like `dismiss`

**Comment by thayashi** [comment]
LGTM

File: src/SuccessToaster/SuccessToasterReducer.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  } = action;
+
+  switch (type) {
+    case SUCCESS:
+      return {
+        ...state,
+        data,
+        title,
+      };
+    case CLEAR_RESPONSE:
```
Comment (stepek) [inline src/SuccessToaster/SuccessToasterReducer.js]: Please rename this action to something like dismiss

**Comment by michal-damiecki** [comment]
@stepek done

**Pull request merged**